package.json

```diff
@@ -13,7 +13,8 @@
 		"node": ">=8"
 	},
 	"scripts": {
-		"test": "xo && ava && tsd"
+		"ava": "ava -s",
```

Comment:
I prefer using the AVA config in package.json to set this instead of command-line flag.